server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/location/LocationDetailExample.java

```diff
@@ -123,4 +125,8 @@ public Integer getTerminalCapacity() {
         return new ArrayList<>(openingHours);
     }
 
+    @Override
+    public List<ICashCollectionDay> getCashCollectionDays() {
+        return new ArrayList<>(cashCollectionDays);
```

Comment:
`cashCollectionDays` is already an ArrayList, why do you need to initialize it again?